bower.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "paper-dialog-behavior",
+  "version": "0.9.0",
+  "authors": "The Polymer Authors",
```

Comment:
During the meeting today it was revealed that the element catalogue is parsing `bower.json` and using the `"description"` field for some things :memo: